tests/EntityControlTraitTest.php

```diff
@@ -196,6 +202,54 @@ public function testCreateOnlyFillable()
         $this->assertSettablePropertiesReset(self::$testRepositoryClass);
     }
 
+    public function testInsert()
+    {
+        $this->mockInsertData();
+
+        $result = self::$testRepositoryClass
+            ->insert([
+                ['name' => 'test_name_1'],
+                ['name' => 'test_name_2'],
+                ['name' => 'test_name_3'],
+            ]);
+
+        $this->assertTrue($result);
+
+        $this->assertSettablePropertiesReset(self::$testRepositoryClass);
```

Comment:
let's test it as a separate test case by calling insert method with the `force` flag